tests/test_transforms/test_encoders/test_categorical_transform.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import pytest
+
+from etna.datasets import TSDataset
+from etna.datasets import generate_periodic_df
+from etna.transforms.encoders.categorical import LabelEncoderTransform
+from etna.transforms.encoders.categorical import OneHotEncoderTransform
+
+
+@pytest.fixture
+def two_df_with_new_values():
+    df1 = TSDataset.to_dataset(
+        generate_periodic_df(periods=3, start_time="2020-01-01", scale=10, period=2, n_segments=2)
+    )
+    df2 = TSDataset.to_dataset(
+        generate_periodic_df(periods=3, start_time="2020-01-01", scale=10, period=3, n_segments=2)
+    )
+    return df1, df2
+
+
+@pytest.fixture
+def df_for_categorical_encoding():
+    return TSDataset.to_dataset(
+        generate_periodic_df(periods=4, start_time="2020-01-01", scale=10, period=3, n_segments=2)
+    )
+
+
+def test_label_encoder(df_for_categorical_encoding):
+    """Test LabelEncoderTransform correct works."""
```

Comment:
I think, that better description is:
"Test that LabelEncoderTransform works correct"